NGit.Test/NGit.Api/ApplyCommandTest.cs

```diff
@@ -78,10 +90,10 @@ private ApplyResult Init(string name, bool preExists, bool postExists)
 			}
 			if (postExists)
 			{
-				b = new RawText(ReadFile(name + "_PostImage"));
+				b = new RawText(ReadFile(name + "_PostImage", a != null && a.GetLineDelimiter() == "\r\n"));
```

Comment:
This is the condition that was very hard to figure out, pretty sure it's right now